app/models/pseud.rb

```diff
@@ -5,7 +5,13 @@ class Pseud < ApplicationRecord
 
   has_attached_file :icon,
     styles: { standard: "100x100>" },
-    path: %w(staging production).include?(Rails.env) ? ":attachment/:id/:style.:extension" : ":rails_root/public:url",
+    path: if Rails.env.production?
+            ":attachment/:id/:style.:extension"
+          elsif Rails.env.staging?
+            "staging/:attachment/:id/:style.:extension"
+          else
+            ":rails_root/public:url"
+          end,
```

Comment:
Does this need to be done for collections/skins, too? Or is that not in scope?

Reply:
Yeah, we need to fix this for collections and skins too, but I think in a separate issue.

Reply:
I made [AO3-6324](https://otwarchive.atlassian.net/browse/AO3-6324) for that.